src/main/scala/nl/knaw/dans/api/sword2/DepositProperties.scala

```diff
@@ -65,8 +65,8 @@ object DepositProperties {
   private def readProperties(f: File) = {
     val ps = new PropertiesConfiguration()
     ps.setDelimiterParsingDisabled(true)
-    ps.load(f)
-
+    if(f.exists) ps.load(f)
```

Comment:
Looked in the `PropertiesConfiguration` source code and this is exactly what we want. Great job! Sadly we need to do this ourselves due to the `setDelimiterParsingDisabled(true)` thingie...